--- src/fltk-config.py	2020-12-30 10:13:11.000000000 +0000
+++ ../p4vasp_mod/src/fltk-config.py	2020-12-30 10:13:40.000000000 +0000
@@ -9,10 +9,10 @@
     os.system("%s >fltk-config.tmp"%cmd)
     return open("fltk-config.tmp").read()
 
-if 0 and strip(run("fltk-config --version")) in ["1.1.5","1.1.6","1.1.7","1.1.8"]:
-    print run("fltk-config "+" ".join(sys.argv[1:]))
-else:
-    if not os.path.exists("../ext/bin/fltk-config"):
-        os.system("cd ../ext; bash build-fltk.sh >build-fltk.log 2>build-fltk.err")
-    print run("../ext/bin/fltk-config "+" ".join(sys.argv[1:]))
-    
+print run("fltk-config "+" ".join(sys.argv[1:]))
+#if 0 and strip(run("fltk-config --version")) in ["1.1.5","1.1.6","1.1.7","1.1.8"]:
+#    print run("fltk-config "+" ".join(sys.argv[1:]))
+#else:
+#    if not os.path.exists("../ext/bin/fltk-config"):
+#        os.system("cd ../ext; bash build-fltk.sh >build-fltk.log 2>build-fltk.err")
+#    print run("../ext/bin/fltk-config "+" ".join(sys.argv[1:]))   
